modules/api/php/endpoints/candidate/visit/dicoms.class.inc

```diff
@@ -153,6 +171,170 @@ class Dicoms extends Endpoint implements \LORIS\Middleware\ETagCalculator
         return $this->_cache;
     }
 
+    /**
+     * Upload the DICOM study file and add it to the timepoint's MRIUploads
+     * A successful upload should return a 303 response toward this DICOM
+     * processes.
+     *
+     * @param ServerRequestInterface $request The incoming PSR7 request
+     *
+     * @return ResponseInterface The outgoing PSR7 response
+     */
+    private function _handlePOST(ServerRequestInterface $request): ResponseInterface
+    {
+        // *************************
+        // *** Permission checks ***
+        // *************************
+        $user = $request->getAttribute('user');
+        if (!$user->hasPermission('imaging_uploader')) {
+            return new \LORIS\Http\Response\JSON\Forbidden();
+        }
+
+        // *************************
+        // *** Input Validation ****
+        // *************************
+        $files = $request->getUploadedFiles();
+        if (count($files) === 0) {
+            return new \LORIS\Http\Response\JSON\BadRequest(
+                'No file found, there should be one file uploaded per request. '
+                    . 'If a file was uploaded, check that it is not larger than '
+                    . 'the max file size PHP setting or take a look at the '
```

Comment:
It is an API user who sees this message, not a LORIS sysadmin. They can not check the max file size PHP setting or look a the Apache logs.

Reply:
Changed the message.
https://github.com/aces/Loris/pull/9154/commits/59437dec438d0ea7a02d7ebfef988b737ad24885

It is less specific but should still be enough for a LORIS dev to fix the error, which is important based on how our tests with Cécile went.